app/src/main/java/com/antosik/benchproject/app/more/view/MoreFragment.kt

```diff
@@ -18,8 +47,134 @@ class MoreFragment : NavigableFragment<MoreFragmentNavigator>() {
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?,
-    ) = MoreFragmentBinding.inflate(inflater, container, false).apply {
-        lifecycleOwner = viewLifecycleOwner
-        viewModel = moreViewModel
-    }.root
+    ) = ComposeView(requireContext()).apply {
+        setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+        setContent {
+            BenchProjectTheme {
+                Column(
+                    modifier = Modifier
+                        .verticalScroll(rememberScrollState())
+                        .padding(bottom = 58.dp)
+                ) {
+                    LabelText(textResourceId = R.string.descriptionMoreFragmentLabel)
+                    DescriptionText(textResourceId = R.string.descriptionMoreFragmentText)
+                    GreyDivider()
+                    LabelText(textResourceId = R.string.licensesMoreFragmentLabel)
+                    TextButtonWithTextResource(
+                        textResourceId = R.string.showLicensesLabel,
+                        onClick = { moreViewModel.onLicensesClick() },
+                    )
+                    GreyDivider()
+                    LabelText(textResourceId = R.string.aboutMeMoreFragmentLabel)
+                    IconAndTextButtonWithTextResource(
+                        textResourceId = R.string.githubLabel,
+                        iconResourceId = R.drawable.ic_github_logo,
+                        onClick = { moreViewModel.onGithubClick() },
+                    )
+                    IconAndTextButtonWithTextResource(
+                        textResourceId = R.string.linkedinLabel,
+                        iconResourceId = R.drawable.ic_linkedin_logo,
+                        onClick = { moreViewModel.onLinkedinClick() },
+                    )
+                }
+            }
+        }
+    }
+
+    @Composable
+    fun LabelText(
+        @StringRes textResourceId: Int,
+    ) {
+        Text(
+            text = stringResource(id = textResourceId),
+            fontSize = 20.sp,
+            textAlign = TextAlign.Center,
+            fontWeight = FontWeight.Bold,
+            color = MaterialTheme.colors.onPrimary,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(top = 16.dp),
+        )
+    }
+
+    @Composable
+    fun DescriptionText(
+        @StringRes textResourceId: Int,
+    ) {
+        Text(
+            text = stringResource(id = textResourceId),
+            fontSize = 16.sp,
+            textAlign = TextAlign.Justify,
+            color = MaterialTheme.colors.onPrimary,
+            modifier = Modifier
```

Comment:
If I remember correctly the `modifier` parameter should be the first one when calling a function with named parameters (but please verify this according to Compose guidelines).

Reply:
I didn't find any place where it will be written about the convention where the modifier should be. I will try to keep the same conventions as in compose method. 
For example:
```
@Composable
fun Text(
    text: String,
    modifier: Modifier = Modifier,
    color: Color = Color.Unspecified,
    fontSize: TextUnit = TextUnit.Unspecified,
    fontStyle: FontStyle? = null,
    fontWeight: FontWeight? = null,
    fontFamily: FontFamily? = null,
    ...
)
```